### PR TITLE
Issue #7450: Error when Jar and Usr specified

### LINKED ISSUE #7450: Update server package to throw error when .jar archive and include=usr are used in combination
The KnowledgeCenter docs for packaging a server state:
The --include=usr option is not valid with an archive format of .jar.

Somehow this check was removed.  Adding it back.

https://www.ibm.com/support/knowledgecenter/en/SS7K4U_liberty/com.ibm.websphere.wlp.zseries.doc/ae/twlp_setup_package_server.html

### PR DESCRIPTION
**Pull request by hlhoots**
fixes #7450 
PR for #7450

### CODE REVIEW

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -700,6 +700,10 @@ error.package.extension=CWWKE0950E: The archive filename must end with .jar when
 error.package.extension.explanation=The --archive option must include a file extension that ends with .jar when specifying --include=runnable.
 error.package.extension.useraction=Supply a file name to --archive with a file extension of .jar when specifying --include=runnable.
 
+error.package.usr.jar=CWWKE0951E: The archive filename must not end with a .jar file extension when the usr option is used.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Change
"The archive **filename** must not end with a .jar file extension when the usr option is used."
to
"The archive **file name** must not end with a .jar file extension when the usr option is used."

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -700,6 +700,10 @@ error.package.extension=CWWKE0950E: The archive filename must end with .jar when
...
+error.package.usr.jar.explanation=The --archive option must not end with a .jar file extension when specifying --include=usr.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Change
"The --archive option must not end with a .jar file extension when **specifying** --include=usr."
to
"The --archive option must not end with a .jar file extension when --include=usr **is specified**."

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 3]
```diff
@@ -700,6 +700,10 @@ error.package.extension=CWWKE0950E: The archive filename must end with .jar when
...
+error.package.usr.jar.useraction=Supply a file name to --archive with a file extension that does not end with .jar when specifying --include=usr.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Change 
"Supply a file name to --archive with a file extension **that does not end with .jar when specifying** --include=usr."
to
"Supply a file name to --archive with a file extension **other than a .jar file extension when** --include=usr **is specified**."

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops (hunk 1 above)
Comment (hlhoots) [reply dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Hey @Charlotte-Holt 

I have fixed all 3 of these and pushed them up.  Thank you for the review!

**Comment by hlhoots** [comment]
#build

**Comment by andrea-pichler** [comment]
Message review completed by L2

**Comment by hlhoots** [comment]
Build was clean.  Merging.

**Pull request merged**